wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsnext/model/PluginMeta.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.workspace.server.wsnext.model;
+
+/** @author Oleksandr Garagatyi */
+public class PluginMeta {
+  private String name = null;
+  private String id = null;
+  private String type = null;
+  private String version = null;
+  private String title = null;
+  private String description = null;
+  private String icon = null;
+  private String url = null;
+
+  public PluginMeta name(String name) {
```

Comment:
why do we need both fluent and then setters ?

Reply:
We probably don't need setters. I just aligned new POJOs with those we already have and latter were originated from auto-generated classes I suppose. But, yeah, we can remove them now.

Reply:
Fixed